outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "cd_webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
+}
```

Comment:
Should be marked as `sensitive = true` so Terraform doesn't print it out, it has your App Service's deployment password in.

<https://app.travis-ci.com/github/CharlieCumber1/DevOps-Course-Starter-Module-3/builds/233145846#L2025>

You can change the password in the Azure portal, go to the App Service -> Container settings (Classic) -> FTPS credentials -> Application scope and reset the password.